packages/terra-notification-dialog/src/NotificationDialog.module.scss

```
@@ -91,15 +91,18 @@
     color: var(--terra-notification-dialog-title-color, #000);
     flex: 0 0 auto;
     flex-wrap: wrap;
-    font-size: var(--terra-notification-dialog-title-font-size, 1rem);
+    font-size: var(--terra-notification-dialog-title-font-size, 1.07142rem);
     font-weight: var(--terra-notification-dialog-title-font-weight, bold);
+    /* stylelint-disable terra/custom-property-name*/
+    margin-top: var(--terra-notification-dialog-title-padding-top, 0.21428rem);
```

I believe this should be:
```suggestion
    margin-top: var(--terra-notification-dialog-title-margin-top, 0.21428rem);
```

---

There's some weirdness here, @neilpfeiffer suggested this be the padding theme variable. Was this to avoid creating a new set of margin values per theme, or a mistake?

---

> There's some weirdness here, @neilpfeiffer suggested this be the padding theme variable. Was this to avoid creating a new set of margin values per theme, or a mistake?

Hey @neilpfeiffer any thoughts on this ?

---

mistake. margin is correct and padding variable name was incorrect.

---

> I believe this should be:

Updated here 23d23edd40f5e5ce50c864566fdd998fdee95882